app/views/appointments/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="text-general">
+
+  <h3>Your appointment #<%= @appointment.id %>:</h3>
+
+  <div>First Name:
+  <%= @appointment.first_name %>
+  </div>
+  <div>Last Name:
```

Comment:
Why would I need to give you my first/last names and email address if I'm already logged in??